docs/source/general/registry.rst

```diff
@@ -0,0 +1,80 @@
+########
+Registry
+########
+
+.. _registry:
+
+********************
+Available Registries
+********************
+
+Registries are mapping a name and metadata to a function.
```

Comment:
This needs to be more clear I'd start by highlighting what registries are for rather than what they are.

Reply:
Done !

Reply:
+1
We could mention more about the available Registries.

Reply:
Done !

Reply:
Added a ref in custom_task.